liver_imaging_analysis/engine/engine.py

```diff
@@ -535,6 +587,64 @@ def predict(self, data_dir):
         return prediction_list
 
 
+    def automate(self,optimization_direction,epochs,evaluate_epochs,batch_callback_epochs,save_weight,save_path,trial_numbers):
```

Comment:
The initialization of this function is similar to the fit function; can you please use the same linting and parameter defaults

 def fit(
        self,
        trial = None,
        epochs = config.training["epochs"],
        evaluate_epochs = 1,
        batch_callback_epochs = None,
        save_weight = False,
        save_path = config.save["potential_checkpoint"],
    ):